src/helpers/certs-helpers.ts

```diff
@@ -15,8 +16,19 @@ export const checkIfCertIsPresent = (certData: string): boolean => {
       encoding: 'utf8',
       flag: 'r',
     });
+    try {
+      const caCert = new X509Certificate(cert);
+      if (caCert.ca) {
+        const certToVerify = new X509Certificate(clientCert.issuerCert.data);
```

Comment:
Can you also put this variable out of the `for` loop since we can define it once and for all?